.travis/assertions.sh

```diff
@@ -28,20 +28,20 @@ testUrlWithSearchString () {
     needle="$2"
     testUrlHttpCode $url
     set -x
-    curl -S -k "$url" | grep -iq "$needle"
+    curl -Ss -k "$url" | grep -iq "$needle"
     set +x
 }
 
 ################################################################################
 # asserts:
 assertLiferayViaNginx () {
     echo "assert that Liferay answers over HTTPS via nginx"
-    testUrlWithSearchString "https://localhost:8443" "<title>Welcome - SW360</title>"
+    testUrlWithSearchString "https://localhost:8443" "Welcome to Liferay Community Edition Portal"
```

Comment:
I've honestly no idea why the general title is returned here. I'm not able to reproduce it on my local machine.
However, since chores is responsible for the infrastructure, I'think it is ok.